democracy_club/templates/contact.html

```diff
@@ -40,10 +40,12 @@ <h2>Contact details</h2>
             <dd><a href="https://www.linkedin.com/company/democracy-club-cic/">LinkedIn</a></dd>
         </div>
         <div>
-            <dt>Twitter</dt>
+            <dt>X</dt>
```

Comment:
Fine, but do we actually want to link to it any more?

Reply:
Yes? At least for the moment. However: I recognise that this leaves 8 in that column and 7 in all the others!

Reply:
Ok